Corvus.Extensions.Samples/TaskExSample.dib

```diff
@@ -0,0 +1,126 @@
+#!markdown
+
+## TaskEx - Sample notebook
+
+#!markdown
+
+First, add a reference to the `Corvus.Extensions` NuGet package.
+
+#!csharp
+
+#r "nuget: Corvus.Extensions, 1.1.4"
+
+#!markdown
+
+Add using statement for `Corvus.Extensions` namespace
+
+#!csharp
+
+using System.IO;
+using Corvus.Extensions;
+using Corvus.Extensions.Tasks;
+
+#!markdown
+
+### WhenAllMany()
+
+This method flattens the outputs from `Task.WhenAll`
+
+#!markdown
+
+#### Examples
+
+The following example uses `WhenAllMany` to asynchronously download all the OPML feeds in a set of feed collections and return a flattened collection of the results.
```

Comment:
My only thought here is that whilst this is clearly a "real world" scenario rather than the more contrived ones we've used elsewhere, that does come at the cost of increased complexity which runs the risk of detracting from the point - showing what the extension method does.

It also adds extra things that could go wrong with the example - for example, if one of your feed endpoints stops working - which is entirely out of our control - then the example will be broken.

Reply:
You could add the endjin  / azure weekly / power bi weekly rss feeds, at least they are under our control

Reply:
> My only thought here is that whilst this is clearly a "real world" scenario rather than the more contrived ones we've used elsewhere, that does come at the cost of increased complexity which runs the risk of detracting from the point - showing what the extension method does.

I agree. We struggled to come up with a basic example and this was simpler than the [GitHub one](https://endjin.slack.com/archives/C02DFDK7W3F/p1663238183724489) being considered.

Any suggestions?

Reply:
>You could add the endjin / azure weekly / power bi weekly rss feeds, at least they are under our control

We could, but there would only be one feed in each of the feed collections in the example. Do you think that would detract from the example?

Reply:
I think you definitely need at least one of the feed collections to include multiple feeds, as otherwise, the point of this operation is lost.

Reply:
> Any suggestions?

No, if you've already attempted to come up with something simpler but not found a good way of doing it, this is fine. You could add some comments to the code to make it easier for people to understand on a single read through.

Reply:
Updated description of `WhenAllMany` method. Added comments describing the example. Updated the feeds used in the example to be endjin controlled ones plus another. https://github.com/corvus-dotnet/Corvus.Extensions/pull/194/commits/0364e0b24da696d879020267dbfa01bcee61e210